grafana-plugin/src/styles/utils.styles.ts

```diff
@@ -26,6 +31,37 @@ export const getUtilStyles = (theme: GrafanaTheme2) => {
     cursorDefault: css`
       cursor: default;
     `,
+
+    overflowChild: css`
+      overflow: hidden;
+      text-overflow: ellipsis;
+      display: -webkit-box;
+      white-space: initial;
+      -webkit-line-clamp: 2;
+      -webkit-box-orient: vertical;
+
+      &--line-1 {
+        -webkit-line-clamp: 1;
+      }
+
+      &--line-3 {
+        -webkit-line-clamp: 3;
+      }
+    `,
+
+    wordBreakAll: css`
+      word-break: break-all;
+    `,
+
+    flex: css`
+      display: flex !important;
+      flex-direction: row;
```

Comment:
`row` is already default direction for flex

Reply:
Right, but that's just to make sure you overwrite it, you know what I mean? Even though that's the default flex-direction, when you apply this util class you wanna make sure it applies all styles (and overwrites anything else). This is not new code, this was already being used in other places